lighthouse-cli/test/cli/run-test.js

```diff
@@ -91,10 +91,15 @@ describe('CLI run', function() {
 
 describe('flag coercing', () => {
   it('should force to array', () => {
-    assert.deepStrictEqual(getFlags(`--only-audits foo chrome://version`).onlyAudits, ['foo']);
+    assert.deepStrictEqual(
+      getFlags('https://www.example.com --only-audits foo chrome://version').onlyAudits, ['foo']);
```

Comment:
why the `https://www.example.com` *and* `chrome://version`? can we just pick one this is def confusing :)

Reply:
Oh, I didn't realize that the first one was meant to be a URL. didn't click, since it's a weird URL. I'll change both to example.